orga/tests/integration/components/routes/authenticated/campaigns/details/profiles-tab-test.js

```diff
@@ -78,15 +73,48 @@ module('Integration | Component | routes/authenticated/campaign/details | profil
       assert.contains('identifiant externe');
       assert.contains('123');
     });
+
+    test('it should display participant certification profile info when shared', async function(assert) {
+      // given
+      const campaign = store.createRecord('campaign', {
+        id: 1,
+        name: 'campagne 1',
+        idPixLabel: 'identifiant externe'
+      });
+
+      const participants = [
+        { 
+          firstName: 'Jane',
+          lastName: 'Doe',
+          sharedAt: new Date(2020, 1, 1),
+          pixScore: 10,
+          certifiable: true,
+          certifiableCompetencesCount: 5,
+        }
+      ];
+      participants.meta = { rowCount: 1 };
+
+      this.set('campaign', campaign);
+      this.set('participants', participants);
+
+      // when
+      await render(hbs`<Routes::Authenticated::Campaigns::Details::ProfilesTab @campaign={{campaign}} @participants={{participants}}/>}}`);
+
+      // then
+      assert.contains('01/02/2020');
+      assert.contains('10');
+      assert.contains('Certifiable');
+      assert.contains('5');
+    });
   });
 
-  module('when there are participants', function() {
+  module('when there aren\'t participants', function() {
```

Comment:
le "personne" est singulier :
```suggestion
  module('when there is no participant', function() {
```